### PR TITLE
Update Point.xml

### PR DESCRIPTION
**Pull request by jrmoreno1**
Structure not a class

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: xml/System.Drawing/Point.xml [hunk 1]
```diff
@@ -84,7 +84,7 @@
         <AssemblyVersion>4.1.0.0</AssemblyVersion>
       </AssemblyInfo>
       <Docs>
-        <summary>Initializes a new instance of the <see cref="T:System.Drawing.Point" /> class with the specified coordinates.</summary>
+        <summary>Initializes a new instance of the <see cref="T:System.Drawing.Point" /> structure with the specified coordinates.</summary>
```
Comment (gewarren) [inline xml/System.Drawing/Point.xml:87]:
```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Drawing.Point" /> struct with the specified coordinates.</summary>
```

File: xml/System.Drawing/Point.xml [hunk 2]
```diff
@@ -119,7 +119,7 @@
       </Parameters>
       <Docs>
         <param name="sz">A <see cref="T:System.Drawing.Size" /> that specifies the coordinates for the new <see cref="T:System.Drawing.Point" />.</param>
-        <summary>Initializes a new instance of the <see cref="T:System.Drawing.Point" /> class from a <see cref="T:System.Drawing.Size" />.</summary>
+        <summary>Initializes a new instance of the <see cref="T:System.Drawing.Point" /> structure from a <see cref="T:System.Drawing.Size" />.</summary>
```
Comment (gewarren) [inline xml/System.Drawing/Point.xml:122]:
```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Drawing.Point" /> struct from a <see cref="T:System.Drawing.Size" />.</summary>
```

File: xml/System.Drawing/Point.xml [hunk 3]
```diff
@@ -166,7 +166,7 @@
       </Parameters>
       <Docs>
         <param name="dw">A 32-bit integer that specifies the coordinates for the new <see cref="T:System.Drawing.Point" />.</param>
-        <summary>Initializes a new instance of the <see cref="T:System.Drawing.Point" /> class using coordinates specified by an integer value.</summary>
+        <summary>Initializes a new instance of the <see cref="T:System.Drawing.Point" /> structure using coordinates specified by an integer value.</summary>
```
Comment (gewarren) [inline xml/System.Drawing/Point.xml:169]:
```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Drawing.Point" /> struct using coordinates specified by an integer value.</summary>
```

File: xml/System.Drawing/Point.xml [hunk 4]
```diff
@@ -225,7 +225,7 @@
       <Docs>
         <param name="x">The horizontal position of the point.</param>
         <param name="y">The vertical position of the point.</param>
-        <summary>Initializes a new instance of the <see cref="T:System.Drawing.Point" /> class with the specified coordinates.</summary>
+        <summary>Initializes a new instance of the <see cref="T:System.Drawing.Point" /> structure with the specified coordinates.</summary>
```
Comment (gewarren) [inline xml/System.Drawing/Point.xml:228]:
```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Drawing.Point" /> struct with the specified coordinates.</summary>
```

File: xml/System.Drawing/Point.xml (hunk 1 above)
Comment (jrmoreno1) [reply xml/System.Drawing/Point.xml]: I think structure would have been fine, but am OK with using the C# keyword instead, it’s a common abbreviation even when talking.

Comment (gewarren) [reply xml/System.Drawing/Point.xml]: Yes, you're probably right, but since the page title uses "Struct", I suggested that.

**Pull request merged**